package.json

```diff
@@ -3,27 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
-    "@types/jest": "^20.0.4",
-    "@types/react": "^15.6.18",
-    "@types/react-native": "^0.52.24",
-    "@types/react-native-vector-icons": "^4.6.1",
-    "@types/react-navigation": "^1.5.12",
-    "babel-jest": "^23.4.2",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-es2016": "^6.24.1",
-    "babel-preset-node6": "^11.0.0",
+    "@types/jest": "^23.3.8",
+    "@types/react": "^16.4.18",
+    "@types/react-native": "^0.57.7",
+    "@types/react-native-vector-icons": "^4.6.4",
+    "@types/react-navigation": "^2.0.24",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "23.6.0",
     "eslint": "^4.19.1",
     "jest": "^23.6.0",
-    "jest-fetch-mock": "^1.6.5",
+    "jest-fetch-mock": "^1.6.6",
+    "metro-react-native-babel-preset": "0.48.3",
     "mocha": "^5.2.0",
-    "mock-async-storage": "^1.0.3",
-    "react-native-debugger-open": "^0.3.17",
```

Comment:
Why did you remove the react-native-debugger-open? I use it to start the standalone debugger.

Reply:
I think there were a few unused modules which I removed, based on what was referenced in the code, I'll re-add this one.